apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: VideoPlayer
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to present videos availible via CDN or local.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {VideoPlayer} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<VideoPlayer title="HLS On Demand Demo" crossOrigin="anonymous">
```

Comment:
`crossOrigin` isn't a required attribute, and I don't think we should be opinionated about the value `anonymous` as end-user CORS expectations could differ. 

```suggestion
<VideoPlayer title="HLS On Demand Demo">
```

Reply:
This was specifically to make the demo work. Ideally, I will have some demo fixtures uploaded to our public folder so we can use those without this attribute.

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/373/commits/6f9a9c8740844ac6d50be622f1c9aab106c564dc)